lib/bounty.rb

```diff
@@ -0,0 +1,250 @@
+class Bounty
+  def self.current
+    Task.new(Parser.parse(checkbounty))
+  end
+
+  # Delegate class methods to current instance of bounty
+  [:status, :type, :requirements, :town, :any?, :none?, :done?].each do |attr|
```

Comment:
There is a lot of ruby going on here. This is code I originally wrote a long time ago and might not write in such a meta way so let me know if it would be helpful to simplify this and expand the method definitions to make them explicit.